pom.xml

```diff
@@ -79,86 +85,44 @@
                         <artifactId>spring-boot-maven-plugin</artifactId>
                         <executions>
                             <execution>
-                                <!-- Spring Boot Actuator displays build-related information if a META-INF/build-info.properties file is present -->
+                                <id>get-the-git-infos</id>
                                 <goals>
                                     <goal>build-info</goal>
                                 </goals>
                                 <configuration>
                                     <additionalProperties>
-                                        <encoding.source>${project.build.sourceEncoding}</encoding.source>
                                         <encoding.reporting>${project.reporting.outputEncoding}</encoding.reporting>
+                                        <encoding.source>${project.build.sourceEncoding}</encoding.source>
                                         <java.source>${maven.compiler.source}</java.source>
                                         <java.target>${maven.compiler.target}</java.target>
                                     </additionalProperties>
                                 </configuration>
                             </execution>
                         </executions>
                     </plugin>
-
-                    <!-- Spring Boot Actuator displays build-related information if a META-INF/build-info.properties file is present -->
                     <plugin>
-                        <groupId>pl.project13.maven</groupId>
-                        <artifactId>git-commit-id-plugin</artifactId>
+                        <groupId>io.github.git-commit-id</groupId>
+                        <artifactId>git-commit-id-maven-plugin</artifactId>
+                        <version>5.0.0</version>
```

Comment:
Why do we need to override the version declared in the Spring Boot parent pom?

Reply:
The version isn't referenced in the parent anymore, as the groupId changed in the plugin v5.x.

Reply:
I'll prefer to keep the artefact `pl.project13.maven:git-commit-id-plugin` from the Spring Boot parent poms :
* https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-starter-parent/2.7.5/spring-boot-starter-parent-2.7.5.pom
* https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.7.5/spring-boot-dependencies-2.7.5.pom
```
<git-commit-id-plugin.version>4.9.10</git-commit-id-plugin.version>
```